src/state.ts

```diff
@@ -75,9 +75,9 @@ export async function getNonProjectRootDir(
 ): Promise<vscode.Uri> {
     let root: vscode.Uri;
     if (!process.env['NEW_DRAMS']) {
-        root = (await context.globalState.get(
+        root = await context.globalState.get<Promise<vscode.Uri>>(
             NON_PROJECT_DIR_KEY
-        )) as vscode.Uri;
+        );
```

Comment:
I'm not 100% on this needing to be an await but things seem functional with the promise/await in place.